partner-portal/src/UI/src/app/get-assistance/cancel-callback-dialog/cancel-callback-dialog.component.html

```diff
@@ -0,0 +1,44 @@
+<div mat-dialog-title class="title">
+  <h2>Cancel Request Call Back</h2>
+  <button mat-icon-button aria-label="Close" mat-dialog-close>
+    <mat-icon>close</mat-icon>
+  </button>
+</div>
+
+<div class="dialog-content">
+  <p>Are you sure you want to cancel your Request Call Back?</p>
+  <div>
+    <mat-card>
+      <mat-card-content class="callbackData">
+        <div style="grid-area: A">
+          <label>Request Callback Date</label>
+          <div>
+            <b>April 28, 2023</b>
```

Comment:
we will have to remember to update the hard-coded values on both this and the shared library pages